Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM php:apache
+MAINTAINER David A. Lareo "dalareo@gmail.com"
+
+RUN apt-get update -y \
+	&& apt-get install -y git libmcrypt-dev libldap2-dev \
+	&& docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ \
+
+	&& docker-php-ext-install mcrypt ldap
+
+RUN git clone https://github.com/simplesamlphp/simplesamlphp.git /var/simplesamlphp
```

Comment:
The problem with this is that it is not really suitable for development, as you don't have direct access to the code that you are running. You would need to stop and start the container again everytime you want to test something, which is why I was suggesting to mount the directory. But again, I don't think that's really suitable for production, hence my question about having two separate Docker files...